BankApplicationPractice-main/BankLibrary/Bank.cs

```diff
@@ -7,41 +7,32 @@ public class Bank<T> where T : Account
     {
         private const string KgkPassPhrase = "CleanUp";
         private readonly List<Account> _accounts = new();
-        private readonly List<Locker> _lockers = new();
+        private readonly Dictionary <Locker, Locker> _accountsDict = new Dictionary<Locker, Locker>();
```

Comment:
не нужно хранить тип локер как значение, локер служит только как ключ
```suggestion
        private readonly Dictionary <Locker, object> _accountsDict = new Dictionary<Locker, object>();
```
а значение это какраз данные пользователя
т.е. в классе локер поле data нужно убрать